foods.html

```diff
@@ -3,9 +3,24 @@
 <head>
   <meta charset="utf-8">
   <title>Quantified Self - Manage Foods</title>
-</head>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script></head>
```

Comment:
i'd add this to your package json and allow webpack to load this for you 👍

Reply:
@case-eee how would I go about adding it to my package.json?